inventory.py

```diff
@@ -1,97 +1,128 @@
+import textwrap
+
 class Order:
     def __init__(self, purchase = {}) -> None:
+        # Unclear naming here. I would assume add_purchase would replace this,
+        # not update this. Even renaming it to `purchases` would be clearer
         self.purchase = purchase
 
     def add_purchase(self, name, quantity):
         self.purchase[name] = quantity
 
-
+# This class was pretty well done, just added some syntax tricks to make it 
+# read cleaner
 class Inventory:
     ALLOWED_ACTIONS = ['1', '2', '3', '4', '5']
 
-    def __init__(self, items = {}) -> None:
+    def __init__(self, items = {}) -> None: 
+        # Providing items here as an optional parameter instead of always initializing
+        # it to an empty dictionary is a good design choice, but you need to be
+        # able to explain why. Why did you choose this?

```

Comment:
I wanted to be able to create an inventory, even if no items were in it yet. An inventory could be empty, but you may still want the inventory object to exist. In rereading this though, I would rather rewrite this to items = none, then add an if statement to prevent multiple inventory instances from sharing the one dictionary:

def __init__(self, items = None) -> None: 
    if items is None: 
        items = {}
    self.items = items

Reply:
No, the way you did it is good, it's just more that you could have designed it _without_ the `items` parameter. It's smart to have the `items` parameter for testing purposes, but I would expect you to be able to justify that decision.

Also, the proposed code is just a wordier version of what you did earlier.

```
if items is None
  items = {}
```

is identical to having the optional `items = {}` parameter.